app.py

```diff
@@ -96,7 +124,33 @@ def sync(dst: str, debug: bool = False, force: bool = False) -> None:
                         proc.returncode,
                         proc.stderr,
                     )
+                    number_unsynced_documents += 1
+                else:
+                    number_synced_documents += 1
```

Comment:
It might be a good idea to have `unsynced`, `synced`, and the ones that were `skipped` if they already exist.

Reply:
added in https://github.com/thoth-station/document-sync-job/pull/6/commits/566b4593121924ea923462677d64d7ef2f57df2f